test/images/image-locality-test/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM BASEIMAGE
+
+CROSS_BUILD_COPY qemu-QEMUARCH-static /usr/bin/
+
+# create a 3GB file filled by zeros
```

Comment:
@ixdy Thanks, I see ... while this test is mean to verify "image locality feature works as expected when image size is large". And ">= 3GB" is a threshold we defined as "large" in the feature which works for all container runtimes.

One approach in my mind is we can use a smaller image (1GB~2GB) if we do not consider containerd case in this test. But I don't know if it's reasonable. WDYT?

If you have other ideas, please feel free to let us know.

Reply:
@ixdy @resouer - an alternative is to reuse the images that already on gcr.io, e.g., the tensorflow image + some other images s.t. the sum of the image sizes is as large as what our use case needs. 

We will need the source file/dockerfile of those images, however.

Reply:
I guess I'm wondering whether this should be an e2e test at all. When will this test be run? Downloading a 3gb image for every e2e test run can be rather expensive, particularly when run outside GCP. And would it cause problems for nodes with smaller disks?

Reply:
I see your point.

@resouer do you think if we e2e test with a small image (≥ 23mb, though the node w/ the image may not receive the maximum priority, as long as the other priority functions do not take over the selection) it would also meet the goal?

I suppose if the testing image can be small, then we should e2e test ImageLocalityPriority.

Reply:
@silveryfu Yes, we can use a smaller image, the logic is right as long as pod can be scheduled to the node with higher score.

Alternative approach is only include integration test for this feature. But I think have a e2e guard for this is preferred.

@ixdy Sounds reasonable if the image is ~100MB?

Reply:
integration test sounds better to me if it gets all of the needed coverage.

Reply:
@ixdy Thanks! We have shipped #68081

@silveryfu Maybe you can close this one as well :D

Reply:
based on the above comment closing this PR